Documentation/CRDs/ceph-client-crd.md

```diff
@@ -34,10 +38,121 @@ metadata:
   namespace: rook-ceph
 spec:
   caps:
-    mon: 'profile rbd'
+    mon: 'profile rbd, allow r'
     osd: 'profile rbd pool=volumes, profile rbd pool=vms, profile rbd-read-only pool=images'
 ```
 
-### Prerequisites
+To use `CephClient` to connect to a Ceph cluster:
 
-This guide assumes you have created a Rook cluster as explained in the main [Quickstart guide](../Getting-Started/quickstart.md).
+### 2. Find the generated secret for the `CephClient`
+
+Once your `CephClient` has been processed by Rook, it will be updated to include your secret:
+
+```console
+kubectl -n rook-ceph get cephclient example -o jsonpath='{.status.info.secretName}'
```

Comment:
The CephClient CR in the example is `sqlitevfs`, correct? 
```suggestion
kubectl -n rook-ceph get cephclient sqlitevfs -o jsonpath='{.status.info.secretName}'
```

Reply:
Ah yes, you're right -- I'll change it there too

Reply:
ended up moving this back to `example` because using the same client for both examples makes a bit less sense naming wise (`sqlitevfs` means nothing to someone that just wants the first section with no intent to use it for the SQLite VFS)